C++/14-Tree/IdenticalTree.cpp

```diff
@@ -0,0 +1,154 @@
+#include <bits/stdc++.h>
```

Comment:
@Bhavyaatrivedi 

Same here.
At the start of every program, we need to write a comment about the program and what problem it is solving. It really helps to know about the program.